src/models/User.js

```diff
@@ -185,8 +185,8 @@ export default (sequelize, DataTypes) => {
   {
     classMethods: {
       associate: models => {
-        User.hasMany(models.Requests, {
-          foreignKey: 'userId',
+        User.hasMany(models.Request, {
+          foreignKey: 'UserId',
```

Comment:
why did you change the foreign key?

Reply:
I changed it because Sequelize throws an error with camel-cased variables.

Reply:
to solve the problem, there is an extra association in the User model. delete it

Reply:
Okay.